packit_service/worker/reporting.py

```diff
@@ -308,7 +306,7 @@ def set_status(
             self.project_with_commit.set_commit_status(
                 self.commit_sha, state_to_set, url, description, check_name, trim=True
             )
-        except gitlab.exceptions.GitlabCreateError as e:
+        except GitlabAPIException as e:
             # Ignoring Gitlab 'enqueue' error
             # https://github.com/packit-service/packit-service/issues/741
             if e.response_code != 400:
```

Comment:
How convenient that it's `.response_code` both in `GitlabCreateError` and `ogr.APIException` :D